README.md

```diff
@@ -379,8 +379,9 @@ nix-env -iA nixpkgs.kubeseal
 The `kubeseal` client can be installed on Linux, using the below commands:
 
 ```bash
-wget https://github.com/bitnami-labs/sealed-secrets/releases/download/<release-tag>/kubeseal-<version>-linux-amd64.tar.gz
-tar -xvzf kubeseal-<version>-linux-amd64.tar.gz kubeseal
+KUBESEAL_VERSION='0.23.0'
```

Comment:
I only have a concern here. We are setting up to 0.23.0 (it is the actual one) but with the next releases the documentation will be confused and older. Could you set this value to something like '<release-tag>'  or '<latest-release-tag>' just to avoid this situation?

Reply:
That's absolutely a valid concern! Stale documentation is no bueno.

I could either change it to a non-value like you suggest, e.g. like `KUBESEAL_VERSION=''` or `KUBESEAL_VERSION='CHANGEME'`.

Or I could expand the example to one where it always fetches the latest version and one where it fetches a pinned version.

Which one do you prefer?

Reply:
KUBESEAL_VERSION='CHANGEME' is a good option. thanks a lot @lindhe

Reply:
I changed my mind. I think `KUBESEAL_VERSION=''` is better than `KUESEAL_VERSION='CHANGEME'`, since the former will be caught by `${KUBESEAL:?}` while the latter will result in `wget` trying to download the `CHANGEME` version. :) Also added an example as a comment to make it clear exactly what the user should set the variable to.